build/prep/Antlr3.Runtime.nuspec

```diff
@@ -28,6 +28,13 @@
     <file src="..\..\Runtime\Antlr3.Runtime\bin\net40-client\$Configuration$\Antlr3.Runtime.pdb" target="lib\net40-client"/>
     <file src="..\..\Runtime\Antlr3.Runtime\bin\net40-client\$Configuration$\Antlr3.Runtime.xml" target="lib\net40-client"/>
 
+    <file src="..\..\Runtime\Antlr3.Runtime\bin\portable-net45\$Configuration$\Antlr3.Runtime.dll" target="lib\portable-net45+win8+wp8+wpa81\" />
+    <file src="..\..\Runtime\Antlr3.Runtime\bin\portable-net45\$Configuration$\Antlr3.Runtime.pdb" target="lib\portable-net45+win8+wp8+wpa81\" />
+    <file src="..\..\Runtime\Antlr3.Runtime\bin\portable-net45\$Configuration$\Antlr3.Runtime.xml" target="lib\portable-net45+win8+wp8+wpa81\" />
+    <file src="..\..\Runtime\Antlr3.Runtime\bin\portable-net45\$Configuration$\Antlr3.Runtime.dll" target="lib\dotnet\" />
```

Comment:
:thought_balloon: It was my understanding that `lib\dotnet\` can only be used when you also have a `<dependencies>` element (or is it `<frameworkDependencies>`?). I would prefer to configure the .NET Core support separately from the addition of a Profile 259 build.